addons_rksv/pos_rksv/static/src/xml/receipt.xml

```diff
@@ -4,25 +4,28 @@
     <!-- Extend the PosTicket - include QRCode - and for testing the OCR Representation -->
     <t t-extend="PosTicket">
         <t t-jquery="t[t-esc='order.name']" t-operation="after">
-            <p>Bon Nr.: <t t-esc="receipt.formatted_receipt_id" /></p>
+            <p t-if="receipt.formatted_receipt_id">Bon Nr.: <t t-esc="receipt.formatted_receipt_id" /></p>
```

Comment:
Well, the one is directly given by the core model and the other one is set during run time at some point, so I assume the config setting is the better way to determine and moreover speaking for itself, what do you think?